cpp/include/cudf/detail/round.hpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cudf/round.hpp>
+
+namespace cudf {
+//! Inner interfaces and implementations
+namespace detail {
+
+/**
+ * @copydoc cudf::round(column_view const&, int32_t, rounding_method,
+ * rmm::mr::device_memory_resource*)
+ *
+ * @param stream CUDA stream used for device memory operations and kernel launches.
+ */
+std::unique_ptr<column> round(
+  column_view const& input,
+  int32_t decimal_places,
+  rounding_method method,
+  rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource(),
```

Comment:
`mr` should go after `stream`. See #5119. I'm in the process of fixing that issue, so let's not add new APIs with them out of order.

Reply:
Done.